### PR TITLE
Np 1384 split testing utils and constants

### PR DESCRIPTION
**Pull request by axthosarouris**
I split the testing utilities for setting up the local database instance and the different constants because I use them for testing in the service module and the handler modules

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -79,15 +79,17 @@ subprojects {
                 limit {
                     counter = 'METHOD'
                     value = 'COVEREDRATIO'
-                    minimum = 0.80
+                    minimum = 0.9
```
Comment (axthosarouris) [inline build.gradle:82]: test coverage becomes 1.000 in next PR.

File: create-doi-request/build.gradle [hunk 2]
```diff
@@ -1,3 +1,27 @@
+repositories {
+    // Repository for running Embedded DynamoDB locally.
+    maven { url "https://s3.eu-central-1.amazonaws.com/dynamodb-local-frankfurt/release" }
+}
+
 dependencies {
     implementation project(":doi-requests-commons")
+    testImplementation(project(":doi-request-common-constants"))
```
Comment (axthosarouris) [inline create-doi-request/build.gradle:8]: database constants used in many modules

File: create-doi-request/build.gradle [hunk 3]
```diff
@@ -1,3 +1,27 @@
...
+    testImplementation( project(":test-module"))
```
Comment (axthosarouris) [inline create-doi-request/build.gradle:9]: testing utilities used in many modules

File: test-module/src/main/java/no/unit/nva/doi/requests/util/DoiRequestsDynamoDBLocal.java [hunk 4]
```diff
@@ -52,7 +56,7 @@ protected void after() {
         }
     }
 
-    protected CreateTableResult createPublicationsTable(AmazonDynamoDB ddb) {
+    protected void createPublicationsTable(AmazonDynamoDB ddb) {
```
Comment (axthosarouris) [inline test-module/src/main/java/no/unit/nva/doi/requests/util/DoiRequestsDynamoDBLocal.java:59]: the result is never used

File: test-module/src/main/java/no/unit/nva/doi/requests/util/DoiRequestsDynamoDBLocal.java [hunk 5]
```diff
@@ -69,12 +73,18 @@ protected CreateTableResult createPublicationsTable(AmazonDynamoDB ddb) {
                 .withGlobalSecondaryIndexes(globalSecondaryIndexes)
                 .withBillingMode(BillingMode.PAY_PER_REQUEST);
 
-        return ddb.createTable(createTableRequest);
+        ddb.createTable(createTableRequest);
+    }
+
+    protected void insertPublication(String tableName, Publication publication) throws JsonProcessingException {
```
Comment (axthosarouris) [inline test-module/src/main/java/no/unit/nva/doi/requests/util/DoiRequestsDynamoDBLocal.java:79]: insert publication directly to the local database. Used during test setup.

File: test-module/src/main/java/no/unit/nva/doi/requests/util/MockEnvironment.java [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+public final class MockEnvironment {
+
+    public static final String ENV_VARIABLE_NOT_FOUND_ERROR = "Did not find env variable:";
+    public static final String ALLOW_CORS = "*";
+
+    /**
+     * Mock environment with all env variables necessary (database and handlers).
+     * @return Environment.
+     */
+    public static Environment mockEnvironment() {
```
Comment (axthosarouris) [inline test-module/src/main/java/no/unit/nva/doi/requests/util/MockEnvironment.java:19]: mock environment used for testing in different modules

**Review by brinxmat — APPROVED** [review]

**Review by janbj — APPROVED** [review]

**Pull request merged**